example/pull/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("//oci:defs.bzl", "oci_image", "structure_test")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+pkg_tar(
+    name = "app",
+    srcs = ["app.bash"],
+)
+
+oci_image(
+    name = "image",
+    architecture = select({
+        "@platforms//cpu:arm64": "arm64",
```

Comment:
Why have select when the values are the same?

Reply:
they are different, `amd64` and `arm64`.